docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,90 @@
+# Scalar DL Backup Creation on Kubernetes
+
+This guide shows you how to create and restore Scalar DL backups on Cosmos DB and DynamoDB without data inconsistencies.
```

Comment:
Does this doc specific for Cosmos and Dynamo?
The title doesn't say so and the contents don't look like so.